### PR TITLE
AutoCreate tables

### PR DESCRIPTION
**Pull request by makss**
AutoCreate tables: tru_tags_prefs and tru_tags_redirects

Fixed errors:
```
A problem occurred while loading the plugin: tru_tags -> User_Error: Table 't_txp.n_tru_tags_prefs' doesn't exist on line 409
```
and
```
User_Error "Table 't_txp.n_tru_tags_redirects' doesn't exist"
in /home/mhosts/repo_txp/txp46dev/textpattern/textpattern/lib/txplib_db.php at line 409.
```

Tested on Txp 4.7-dev

### CODE REVIEW

File: tru_tags.php [hunk 1]
```diff
@@ -796,12 +796,6 @@ function tru_tags_admin_update_prefs() {
 
 
 function tru_tags_upsert_pref($name, $value) {
-	if (!safe_query('create table if not exists ' . safe_pfx('tru_tags_prefs').
-		' (name varchar(255) primary key, '.
-		'value varchar(255) not null)')) {
-		return 'Serious error - unable to create the ' . safe_pfx('tru_tags_prefs') . ' table.';
-	}
-
```
Comment (truist) [inline tru_tags.php:804]: You removed this bit but didn't replace it (here) with anything else. Are you sure that's safe to do?

File: tru_tags.php [hunk 2]
```diff
@@ -849,12 +843,6 @@ function tru_tags_admin_redirect_tag($lefttag, $righttag) {
 		return 'Please enter a value in both fields';
 	}
 
-	if (!safe_query('create table if not exists ' . safe_pfx('tru_tags_redirects').
-		' (lefttag varchar(255) primary key, '.
-		'righttag varchar(255) not null)')) {
-		return 'Serious error - unable to create the ' . safe_pfx('tru_tags_redirects') . ' table.';
-	}
-
 	if (safe_insert('tru_tags_redirects', 'lefttag="'.$lefttag.'",righttag="'.$righttag.'"')) {
```
Comment (truist) [inline tru_tags.php:846]: You removed this bit but didn't replace it (here) with anything else. Are you sure that's safe to do?

File: tru_tags.php (hunk 1 above)
Comment (makss) [reply tru_tags.php:804]: Checking and creating a preference table is done at the [beginning](https://github.com/makss/tru-tags/blob/7108872118ba30d5acbeacc87d4fc961ed674114/tru_tags.php#L67) of the plugin, that is, before the function call `tru_tags_upsert_pref()`.

File: tru_tags.php (hunk 2 above)
Comment (makss) [reply tru_tags.php:846]: I think that everything is in order here. Checking and creating a table occurs when you open the admin tab.

**Pull request merged**